gst-plugins-base.yaml

```diff
@@ -1,7 +1,7 @@
 package:
```

Comment:
Any context for the changes to this existing package? was there build issues before or were we skipping some components?

Reply:
Yes, the gl plugins are needed for the gtk-4. Previously we were not including it but now we are.